### PR TITLE
Fix version for CBRAIN compatibility

### PR DESCRIPTION
**Pull request by natacha-beck**

### CODE REVIEW

File: cbrain_task_descriptors/ndmg-docker.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "ndmg",
-  "tool-version": "v0.0.48-1",
+  "tool-version": "v0.0.48-1d",
```
Comment (gkiar) [inline cbrain_task_descriptors/ndmg-docker.json]: docker image is pushed- we can now make this `v0.0.50d`

File: cbrain_task_descriptors/ndmg-singularity.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "ndmg",
-  "tool-version": "v0.0.48-1",
+  "tool-version": "v0.0.48-1s",
```
Comment (gkiar) [inline cbrain_task_descriptors/ndmg-singularity.json]: docker image is pushed- we can now make this `v0.0.50s`

**Review by gkiar — CHANGES_REQUESTED** [review]
ready to update versions now! want to do that before I accept PR?

**Review by gkiar — APPROVED** [review]

**Comment by natacha-beck** [comment]
Done.

**Pull request merged**